buildSrc/subprojects/plugins/src/main/kotlin/org/gradle/plugins/performance/PerformanceTestPlugin.kt

```diff
@@ -103,6 +106,43 @@ class PerformanceTestPlugin : Plugin<Project> {
         tasks.register("performanceReport")
     }
 
+    private
+    fun Project.registerForkPointDistributionTask() {
+        val buildForkPointDistribution = tasks.register("buildForkPointDistribution", BuildForkPointDistribution::class) {
+            dependsOn("determineForkPoint")
+        }
+        tasks.register("determineForkPoint") {
+            doLast {
+                val masterForkPointCommit = execAndGetStdout("git", "merge-base", "origin/master", "HEAD")
+                val releaseForkPointCommit = execAndGetStdout("git", "merge-base", "origin/release", "HEAD")
+                val forkPointCommit =
+                    if (exec { isIgnoreExitValue = true; commandLine("git", "merge-base", "--is-ancestor", masterForkPointCommit, releaseForkPointCommit) }.exitValue == 0)
+                        releaseForkPointCommit
+                    else
+                        masterForkPointCommit
+                buildForkPointDistribution.get().forkPointCommitId = execAndGetStdout("git", "rev-parse", "--short", forkPointCommit)
+            }
+        }
+        tasks.register("configurePerformanceTestBaseline") {
+            dependsOn(buildForkPointDistribution)
+            doLast {
+                val commitBaseline = (project.tasks.findByName("buildForkPointDistribution") as BuildForkPointDistribution).baselineVersion
+                project.tasks.withType(DistributedPerformanceTest::class) {
+                    if (true == baselines?.isNullOrEmpty() || baselines == "defaults") {
```

Comment:
Can this be simplified to
```
if (baselines.isNullOrEmpty() || baselines == "defaults)
```
?

Reply:
Doesn't `baselines.isNullOrEmpty()` return `Boolean?`?